test/integration_test/utils/node.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"testing"
+
+	coreops "github.com/portworx/sched-ops/k8s/core"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+// AddLabelToRandomNode adds the given label to a random worker node
+func AddLabelToRandomNode(t *testing.T, key, value string) string {
```

Comment:
so this seems to be the same as node affinity test and could be used there as well. But it doesn't add label to a random node but only the first one.

Reply:
It is already being used there. Calling it random because there is no guarantee which node would be the first one. It will depend on what k8s api returns.